src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag

```diff
@@ -62,7 +62,8 @@
                         <br>
                         <div>
                             <f:feedbackSessionPublishButton buttonType="${sessionPanel.feedbackSessionPublishButton.buttonType}"
```

Comment:
`buttonType="btn ${sessionPanel.feedbackSessionPublishButton.buttonType}"` can be used instead so that the if condition in the class of  `<a>` of feedbackSessionPublishButton.tag can be removed.